v2.0/select.md

```diff
@@ -27,7 +27,7 @@ The user must have the `SELECT` [privilege](privileges.html) on the table.
 | `AS col_label` | In the retrieved table, change the column label to `col_label`. |
 | `table_ref` | The [table expression](table-expressions.html) you want to retrieve data from.|
 | `index_name` | The name of the index you want to use, also known as "[index hints](#force-index-selection-index-hints)." Find index names using [`SHOW INDEX`](show-index.html). <br/><br/>Forced index selection overrides [CockroachDB's index selection](https://www.cockroachlabs.com/blog/index-selection-cockroachdb-2/). |
-| `AS OF SYSTEM TIME timestamp` | Retrieve data as it existed as of [`timestamp`](timestamp.html).<br/><br/>For more information, see [this example](#select-historical-data-time-travel) or our blog post [Time-Travel Queries](https://www.cockroachlabs.com/blog/time-travel-queries-select-witty_subtitle-the_future/). |
+| `AS OF SYSTEM TIME timestamp` | Retrieve data as it existed as of [`timestamp`](timestamp.html).<br/><br/>For more information, see [this example](#select-historical-data-time-travel) or our blog post [Time-Travel Queries](https://www.cockroachlabs.com/blog/time-travel-queries-select-witty_subtitle-the_future/).<br/><br/>Because `AS OF SYSTEM TIME` returns historical data, your reads might be stale. |
```

Comment:
I think it'd be helpful to cross-reference the new section in `performance-best-practices-overview.md`.

Reply:
Also, can you mention this use case in the example itself and link to `performance-best-practices-overview.md` from there, too?

Reply:
Done.